src/hotspot/share/runtime/safepointMechanism.cpp

```diff
@@ -113,6 +113,7 @@ uintptr_t SafepointMechanism::compute_poll_word(bool armed, uintptr_t stack_wate
 }
 
 void SafepointMechanism::update_poll_values(JavaThread* thread) {
+  assert(thread == Thread::current(), "Must not be");
```

Comment:
I assume the message should say "Must be".

Reply:
Yes, fixing.